### DISCUSSION TITLE
Audio: Glitchy playback and rendering

### DISCUSSION THREAD

**Discussion by artsyhugh** [opentoonz/opentoonz#6395] [General]
I don't know if it's because I updated my audio drivers or something but now the audio playback is riddled with audio glitches and they still remain in the rendered output video. This doesn't happen with any other video rendering program I use, apart from Toonz Premium which I'm trying and which is having the same problem. Man, I really wished OpenToonz would work for me but it keeps getting my hopes down.

Here's my audio specs:
Intel Display Audio 10.22.1.97
Realtek Audio 6.0.1.8142

**Comment by artisteacher** [comment]
What OS & what version of OT are you using?

**Comment by artsyhugh** [comment]
I'm on Windows 10 and was using the current version of OpenToonz. Are there any other specifications I should list here?

**Comment by artsyhugh** [comment]
FYI, here's a screen recording in which I was trying Toonz Premium, but the issue is the same with OpenToonz even when I was working with an exposed level of sequential images.
https://drive.google.com/open?id=0B_rBg1NdAW2tSFlsd1pHQnN5LUl0djVNRUZBbFp3aVc1eDU4

**Comment by artisteacher** [comment]
Are there any audio files that play without distortion? I've had success in OS X with 16bit wav/mp3/aif. In older versions of OS X, both mono and stereo files play fine. In the latest version of OS X, stereo files play the best.

Does the audio sound ok when playing from the column header?

**Comment by artsyhugh** [comment]
It happens with all of my WAVs. It appears to coincide with whenever the music gets loud enough accordingly to the waveform. As you can see in the example recording above, the music was pretty loud. I've tried with a couple of different songs with more diverse amplitudes, and the glitch doesn't seem to occur when the music is quiet enough.

**Comment by RodneyBaker** [comment]
There are several reports of audio distortions etc. and we'll try to narrow down the scope a little.
In the OpenToonz forum @NixiePinnison commented on the same issue.

[Link to forum topic](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!category-topic/opentoonz_en/PXbCw24XV4c)

**Comment by RodneyBaker** [comment]
I have noticed that when I leave a blank cell in front of the audio (and/or one blank/empty cell after the audio) it allows for clear playback in OpenToonz.  I haven't look much more than that into it but that might resolve some of the problem in that it seems to release the audio so that when it starts up again... for say... looping... the sound plays back nicely. 

@artsyhugh It is interesting that you note the problem in Toonz Premium as well.

**Comment by RodneyBaker** [comment]
Many updates and fixes have occurred to audio in Opentoonz since this report was last discussed (circa 2018).
@artsyhugh It will be good to know if you are still experiencing the same (or similar) glitchy playback (and rendering) when using the most recent nighly releases.

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore information and consider the best way forward specifically with audio in Opentoonz.
